grid2op/multi_agent/test_ma.py

```diff
@@ -49,12 +49,35 @@ def test_verify_domains(self):
         observation_domains = action_domains
         try:
             MultiAgentEnv(self.env, observation_domains, action_domains)
+            assert False
+            
+        except DomainException :
+            assert True
```

Comment:
Whaaaaat is that ? No no no, raise proper error and have a proper handling mechanism. What's the point to assert True or False ?

Reply:
You can use [self.assertRaises](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) for example